tests/Feature/Api/UserTaste/UserTasteTest.php

```diff
@@ -58,12 +61,12 @@ public function test_user_delete_taste() {
             'user_id' => $this->user->id,
```

Comment:
No need to assert it. ` $this->user->tastes()->save($taste);` this is tested internally by Laravel.